NEWS.md

```diff
@@ -204,7 +204,7 @@ to specify the unit of the input age (#569)
 
 ### OCCDS
 
-- Last Dose Date(time) can be derived using `derive_last_dose()`
+- Last Dose Date(time) can be derived using `derive_var_last_dose_date()`
```

Comment:
You have added the changes to admiral 0.2.0. Could you please move them to the latest version. If you merge in develop, you should get a section for the latest version.

Reply:
reverted the note